src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/EnvironmentSetupHelper.cs

```diff
@@ -589,4 +589,4 @@ private void SetupPowerShellModules(System.Management.Automation.PowerShell powe
         }
 
     }
```

Comment:
Please check your git client settings - there are a bunch of weird diffs here which may indicate that you are using the wrong line endings.